.test/builds.json

```diff
@@ -2716,5 +2716,61 @@
         }
       }
     }
+  },
+  "93476ae64659d71f4ee7fac781d6d1890df8926682e2fa6bd647a246b33ad9bf": {
+    "buildId": "93476ae64659d71f4ee7fac781d6d1890df8926682e2fa6bd647a246b33ad9bf",
+    "build": {
+      "img": "oisupport/staging-amd64:93476ae64659d71f4ee7fac781d6d1890df8926682e2fa6bd647a246b33ad9bf",
+      "resolved": null,
```

Comment:
With https://github.com/docker-library/official-images/commit/795e04932a2ed061d24262c83560ab5a4e08d2ef, this is now even safer as part of our tests because this exact `buildId` will never be officially built. :smile: :innocent: